packages/expo/migrations.json

```diff
@@ -17,6 +17,12 @@
       "cli": "nx",
       "description": "Add target build and build-list for expo projects in project.json",
       "factory": "./src/migrations/update-14-4-3/add-eas-build-target"
+    },
+    "add-update-target-14-4-4": {
+      "version": "14.4.4-beta.0",
```

Comment:
could yo change this to "14.5.1-beta.0"? could you update the folder `update-14-4-4` as well?